tests/ttnn/unit_tests/operations/test_math.py

```diff
@@ -252,6 +252,20 @@ def run_math_unary_test_recip(device, h, w, ttnn_function, pcc=0.9999):
     assert_with_pcc(torch_output_tensor, output_tensor, pcc)
 
 
+def run_math_unary_test_fixed_val(device, h, w, fill_value, ttnn_function, pcc=0.9999):
+    torch.manual_seed(0)
+    torch_input_tensor = torch.full((h, w), fill_value, dtype=torch.bfloat16)
+    golden_function = ttnn.get_golden_function(ttnn_function)
+    torch_output_tensor = torch.nan_to_num(
+        golden_function(torch_input_tensor), nan=6.9752e19, posinf=1.6948e38, neginf=-1.6948e38
```

Comment:
The torch.nan_to_num should be applied within the golden function defined in ttnn directly.   The intent is for the golden function itself to return the expected output so that it can continue to be used in the Visualizer.

Reply:
Updated the golden function in `unary.py`